mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelWrapper.kt

```diff
@@ -0,0 +1,10 @@
+package com.airbnb.mvrx
+
+import androidx.lifecycle.ViewModel
+
+class MvRxViewModelWrapper<VM : BaseMavericksViewModel<S>, S : MvRxState>(val viewModel: VM) : ViewModel() {
```

Comment:
as  part of 2.0 release do we want to separate jetpack dependencies from `BaseMavericksViewModel` in separate modules just so if we ever want to support multiplatform we don't have to have breaking changes again? would be nice to get all our module names solidified for that

Reply:
@elihart Agreed!